app/controllers/register/trustees/AddTrusteeController.scala

```diff
@@ -88,5 +88,5 @@ class AddTrusteeController @Inject()(
     isHnSEnabled || trusteeList.forall(_.isCompleted)
   }
 
```

Comment:
Why does this need to be a def?

Reply:
If you change the toggle dynamically using test routes it doesn't change unless this is a def